src/svg/axis.js

```diff
@@ -123,6 +123,15 @@ export default function() {
 
             g.select('text')
                .attr('transform', translate(0, labelOffset))
+               .attr('dy', function() {
+                   var offset = '0em';
```

Comment:
Normally I'm against any magic numbers but as we've gone round on this multiple times I'm willing to let it pass in this instance. Before I do though, can I check where these numbers came from because they don't appear to be symmetrical  (e.g. 0.32/0.68)?